h/activity/bucketing.py

```diff
@@ -48,7 +49,7 @@ def incontext_link(self, request):
         """
         if not self.annotations:
             return None
-        return links.incontext_link(request, self.annotations[0])
+        return incontext_link(request, self.annotations[0])
```

Comment:
We often used these functions directly, keep it as functions but moved them to the service namespace.